{"name": "Проверка невозможности создания двух одинаковых пользователей", "status": "passed", "description": "При создании, используются все данные уже созданного пользователя", "start": 1725449638038, "stop": 1725449639810, "uuid": "cbb2bf35-91be-4c4b-b908-c298608e30b6", "historyId": "c6890ad6f33041df8e1d731cd4a771c6", "testCaseId": "c6890ad6f33041df8e1d731cd4a771c6", "fullName": "test_create_user.TestCreateUser#test_create_user_existent_user_error", "labels": [{"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "DESKTOP-6NP8JLT"}, {"name": "thread", "value": "31440-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_user"}]}